### PR TITLE
Fixing missing requires and state request

### PR DESCRIPTION
**Pull request by cdberkstresser**
This fixes issue #1

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <?php
 require('../bootstrap.php');
+require('../src/services/OktaApiService.php');
```
Comment (aaronpk) [inline public/index.php:3]: These are brought in via `bootstrap.php` with Composer.

File: src/services/OktaApiService.php [hunk 2]
```diff
@@ -28,6 +28,7 @@ public function buildAuthorizeUrl($state)
             'client_id' => $this->clientId,
             'redirect_uri' => $this->redirectUri,
             'state' => $state,
+            'scope' => 'openid',
```
Comment (aaronpk) [inline src/services/OktaApiService.php:31]: Thanks for this!

**Review by aaronpk — CHANGES_REQUESTED** [review]
Thanks for this! The `require` lines shouldn't be necessary though since that's handled through Composer.

File: src/services/OktaApiService.php (hunk 2 above)
Comment (cdberkstresser) [reply src/services/OktaApiService.php:31]: @aaronpk I thought it should work too, but here is what happens without the two requires even after a `composer install`.  Here is the output of `composer install` and `php -S 127.0.0.1:8080 -t public` on Ubuntu 20.04 and PHP 7.4.3:
```
cberkstresser@apollo:~/okta-php-core-login-example$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
cberkstresser@apollo:~/okta-php-core-login-example$ php -S 127.0.0.1:8080 -t public
[Mon Aug 31 12:03:14 2020] PHP 7.4.3 Development Server (http://127.0.0.1:8080) started
[Mon Aug 31 12:03:23 2020] 127.0.0.1:38286 Accepted
[Mon Aug 31 12:03:24 2020] PHP Fatal error:  Uncaught Error: Class 'Src\Services\OktaApiService' not found in /home/cberkstresser/okta-php-core-login-example/public/index.php:7
Stack trace:
#0 {main}
  thrown in /home/cberkstresser/okta-php-core-login-example/public/index.php on line 7
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38286 [500]: GET / - Uncaught Error: Class 'Src\Services\OktaApiService' not found in /home/cberkstresser/okta-php-core-login-example/public/index.php:7
Stack trace:
#0 {main}
  thrown in /home/cberkstresser/okta-php-core-login-example/public/index.php on line 7
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38286 Closing
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 Accepted
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 [404]: GET /favicon.ico - No such file or directory
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 Closing
```

Comment (aaronpk) [reply src/services/OktaApiService.php:31]: okay let me take a closer look...

**Comment by aaronpk** [comment]
Okay I figured out what was happening. The folders that those files were in were lowercase, but the class names are caps, and on Mac this doesn't matter because the filesystem isn't case sensitive. I just pushed a commit that renames the folders to the proper case, and that should fix it: 5adecfb143908c13e40a451deb94e50a2e445c87

**Comment by aaronpk** [comment]
Okay I ended up pushing up the `scope` fix too,  so I'm just going to close this out. Thanks for bringing this up!

**Pull request closed without merging**